conformance/utils/suite/experimental_suite.go

```diff
@@ -142,6 +142,10 @@ func NewExperimentalConformanceTestSuite(s ExperimentalConformanceOptions) (*Exp
 		}
 	}
 
+	for feature := range s.ExemptFeatures {
+		s.SupportedFeatures.Delete(feature)
+	}
```

Comment:
Are `ExemptFeatures` accounted for in display of conformance report? Just want to make sure we're not opening a way to have misleading conformance reports here.

Reply:
I can add another code block in here https://github.com/kubernetes-sigs/gateway-api/blob/6d0e6ccbccc73def46bce2e52ed50ca1e480a890/conformance/utils/suite/experimental_suite.go#L239 which sets the `testSkipped` if the test is an exempt features test

Reply:
Maybe it would be fair to just ensure `exemptFeatures` are added to `unsupportedFeatures` list? Example: https://github.com/kubernetes-sigs/gateway-api/blob/9c830e50acbf0844370210c8f54354b4e062f768/conformance/reports/v0.8.0/nginxinc-nginx-gateway-fabric.yaml#L27-L37

I don't know if exemptFeatures are somehow different than unsupportedFeatures though. Maybe those concepts should just be merged altogether?

Reply:
I also think its not required, because the test is not supposed to run
cc @mlavacca

Reply:
+1 on merging `exemptedFeatures` into `unsupportedFeatures`. A potential problem that comes to my mind is that the core features for the requested profiles should not be exempted. For example, when setting the HTTP profile, it should be disallowed exempting  the`HTTPRoute`, `Gateway`, and `ReferenceGrant` features.

Reply:
updated the PR with above suggestion, ptal @mlavacca @robscott